docs/DeveloperGuide.md

```diff
@@ -50,7 +50,7 @@ There would be 4 settings for the priority feature: High, Medium, Low, Undefined
 
 ![Proposed UI](images/UIPriority.png)
 
-Firstly, the field would be added to the Person class. This would be similar to the method described in the tutorial:  https://nus-cs2103-ay2021s1.github.io/tp/tutorials/AddRemark.html, where instead of adding a command, we would be editing the add command to include an optional priority field. 
+Firstly, the field would be added to the Person class. This would be similar to the method described in the [tutorial](https://nus-cs2103-ay2021s1.github.io/tp/tutorials/AddRemark.html), where instead of adding a command, we would be editing the add command to include an optional priority field. 
```

Comment:
Are we keeping the AddRemark tutorial, and did we maintain it? If it is then it's okay

Reply:
Hmmm it will still be inside. But ok I guess ill remove it. Cuz if we maintain it, all the images and code(code snippets) and everything has to be changed. Too much work

Reply:
Add in p use case

Reply:
Approving this first but removing tutorial is pending

Reply:
Approving this first but priority is pending